src/client.py

```diff
@@ -0,0 +1,88 @@
+from numbers import Number
+
+from open_feature import OpenFeature
+from src.flag_type import FlagType
+
+
+class OpenFeatureClient:
+    def __init__(self, name: str, version: str, context, hooks: list = []):
+        self.name = name
+        self.version = version
+        self.context = context
+        self.hooks = hooks
+
+    def get_boolean_value(
+        self, key: str, defaultValue: bool, evaluationContext, flagEvaluationOptions
```

Comment:
We should use snake case for all method arguments unless there is a specific reason not to?

Reply:
Using snake case everywhere now